### DISCUSSION TITLE
运行错误

### DISCUSSION THREAD

**Discussion by kkchau-ctflife** [alibaba/jetcache#929] [Q&A]
![Snipaste_2024-09-23_23-15-05](https://github.com/user-attachments/assets/3930c80d-497d-450f-a0c3-8d9acf9f9c6b)

![Snipaste_2024-09-23_23-20-12](https://github.com/user-attachments/assets/144504b3-ef23-4705-bb2e-30ca8455466e)

你好，请教一个问题就是我直接运行项目会报错，于是参考源码写了一个测试代码，当去掉条件注解的时候i就能正常运行，是我的配置是哪里出问题了吗

**Comment by areyouok** [comment]
按samples目录下的例子来
